django/middleware/csrf.py

```diff
@@ -33,6 +33,7 @@
 CSRF_SECRET_LENGTH = 32
 CSRF_TOKEN_LENGTH = 2 * CSRF_SECRET_LENGTH
 CSRF_ALLOWED_CHARS = string.ascii_letters + string.digits
+CSRF_SESSION_KEY = 'csrftoken'
```

Comment:
Done. Shall I squash this into one commit now or will that be done while merging?